server/engine/src/main/antlr4/org/eclipse/lsp/cobol/implicitDialects/cics/CICSParser.g4

```diff
@@ -594,7 +594,8 @@ cics_resetbr: RESETBR cics_file_name (RIDFLD cics_data_area | KEYLENGTH cics_dat
               SYSID cics_data_area | GTEQ | EQUAL | RBA | RRN |XRBA | cics_handle_response)+;
 
 /** RESUME */
-cics_resume: RESUME (ACQACTIVITY | ACQPROCESS | ACTIVITY cics_data_value | cics_handle_response)+;
+cics_resume: RESUME (cics_resume_body | cics_handle_response)+;
+cics_resume_body: ((ACQACTIVITY | ACQPROCESS) | ACTIVITY cics_data_value | cics_handle_response)+;
```

Comment:
```suggestion
cics_resume: RESUME (cics_resume_body | cics_handle_response)+;
cics_resume_body: ACQACTIVITY | ACQPROCESS | ACTIVITY cics_data_value;
```

would this work ?

Reply:
That could work, but it might also mess with the checks.

Reply:
This should be cleaned up. Add tests to verify that the checks work correctly if you suspect issues.